src/app/find/locationDetails/CategoryCard.jsx

```diff
@@ -1,150 +1,40 @@
-  const isScheduleKnown = schedule && schedule.length;
-
-  let isClosed = false;
-  let openDays;
-  if (isScheduleKnown) {
-    openDays = schedule.filter(({ closed }) => !closed);
-
-    if (!openDays.length) {
-      isClosed = true;
-    }
+  handleCollapse = () => {
+    this.setState({ collapsed: !this.state.collapsed });
   }
 
-  return (
-    <div
-      key={service.id}
-      style={{
-        borderBottom: isLastItem ? '0' : '1px solid #DADADA',
-        paddingBottom: '1.3vh',
-      }}
-    >
-      <div size="medium" className="specificServiceHeaders">
-        {service.name || service.Taxonomies[0].name}
-        {isClosed && <span className="coronavirusInfo"> (suspended)</span>}
-      </div>
-
-      {!!(coronavirusInfo && coronavirusInfo.length) && (
-        <InfoItem coronavirus icon="exclamation-triangle">
-          {coronavirusInfo[0].information}
-        </InfoItem>
-      )}
+  render() {
+    const { category, services, isSearchCategory } = this.props;
 
-      {!isClosed && (
-        <Fragment>
-          {!!service.description && (
-            <div className="serviceDescription">
-              {service.description}
-            </div>
-          )}
-
-          <ServiceRestrictions
-            eligibilities={service.Eligibilities}
-            requiredDocuments={service.RequiredDocuments}
+    return (
+      <div className="categoryCard">
+        <div className="serviceCategoryHeadersContainer">
+          <Icon
+            name={getCategoryIcon(category)}
+            size="lg"
           />
-
-          {openDays && (
-            <InfoItem coronavirus icon="clock">{renderSchedule(openDays)}</InfoItem>
-          )}
-
-          {service.Phones && service.Phones.map(phone => (
-            <InfoItem key={phone.id} icon="phone">
-              <PhoneLink {...phone} className="locationLinks" />
-            </InfoItem>
-          ))}
-
-          <ServiceOfferings attributes={service.ServiceTaxonomySpecificAttributes} />
-        </Fragment>
-      )}
-    </div>
-  );
-};
-
-const CategoryCard = ({ category, services, className }) => (
-  <div
-    className="categoryCard"
-    style={{
-      backgroundColor: '#F8F8FC',
-      border: '1px solid #DADADA',
-    }}
-  >
-    <div className="serviceCategoryHeadersContainer">
-      <div className="serviceCategoryHeaders">{category}</div>
-      <Icon
-        name={getCategoryIcon(category)}
-        size="2x"
-      />
-    </div>
-
-    {services
-      .sort((service1, service2) => service2.lastUpdate - service1.lastUpdate)
-      .map((service, i) => renderService(service, i === services.length - 1))}
-  </div>
-);
+          <div className="serviceCategoryHeaders">{category}</div>
+        </div>
+        <div>
+          {services
+            .sort((service1, service2) => service2.lastUpdate - service1.lastUpdate)
+            .map(service => <ShelterService service={service} selectedCategory={false} key={service.id} showOnMount={isSearchCategory} />)}
```

Comment:
`selectedCategory` isn't actually used (which makes sense, as any prop that can only get 1 value isn't likely to be needed).